rpa-home/modules/ROOT/pages/automation-userrolesandpermissions-privilegesbymodule.adoc

```diff
@@ -0,0 +1,358 @@
+
+|*Process Recording*
+|Using the RPA Process Recorder
+
+|===
+
+
+
+
+== Privileges of the *Process Operations* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Process Execution Plans Open*
+|Open the *Process Execution Plans* view.
+
+|*Global Process Execution Plans View*
+|Authorization to view all processes in the *Process Execution Plans* view, independently of whether or not the current user is a member of the process team
+
+|*Process Deployment Map Open*
+|Open the view *Deployment Map*
+
+|*Upcoming Process Changes Open*
+|Open the *Upcoming Process Changes* view.
+
+|*Upcoming Process Changes Administration*
+|Administer objects in the *Upcoming Process Changes* view
+
+|*Finance Analysis Open*
+|Open the *Finance Analysis* view with processes for which the current user is a member of the process team
+
+|*Global Finance Analysis View*
+|Open the *Finance Analysis* view with all processes, independently of whether the current user is a member of the process team
+
+|*Finance Analysis Edit*
+|Changing finance analyses
+
+|===
+
+
+
+
+== Privileges of the *Process Monitoring* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Process Monitoring Open*
+|Open all views of the *Process Monitoring* module
+
+|*Robot State and Operation Open*
+|Open the *Robot State and Operation* view
+
+|*Process Streaming Open*
+|Open the *Process Streaming* view
+
+|===
+
+
+
+
+== Privileges of the *My RPA* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Global Process Evaluation View*
+|Authorization to view all processes in the views *Process Evaluation* and *My RPA* &gt; *Team Backlog*, regardless of whether or not the current user belongs to the Process Team.
+
+|*Unprocessed Task List Open*
+|Open the *Unprocessed Tasks* view.
+
+|*Unprocessed Task List Edit*
+|Change the *Unprocessed Tasks* view.
+
+|===
+
+
+
+
+== Privileges of the *Dashboard* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Dashboard Open*
+|Open all views of the *Dashboard* module
+
+|*Dashboard Administration*
+|Create, change, and delete all objects of the *Dashboard* module
+
+|===
+
+
+
+
+== Privileges of the *Alerting* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Alerting Open*
+|Open all views of the *Alerting* module
+
+|*Alerting Administration*
+|Create, change, and delete all objects of the *Alerting* module
+
+|===
+
+
+
+
+== Privileges of the *RPA Builder* category
+
+The *Activity Library* is managed exclusively in the RPA Builder.
+
+Learn how to xref:rpa-builder::ui-overview.adoc#explorer-pane[find the Activity Library in RPA Builder].
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Activity Library Administration*
+|Authorization to manage the *Activity Library* in the RPA Builder.
+
+|*Activity Library Open*
+|Authorization to open the *Activity Library* in the *X1 Design Studio*.
```

Comment:
This should probably also mention RPA builder now

Reply:
Checked for further X1s in the docs-rpa repo, but didn't find any...